cms/static/js/views/content_group_details.js

```diff
@@ -66,31 +68,12 @@ define([
                         Translators: 'count' is number of units that the group
                         configuration is used in.
                     */
-                    'Used in %(count)s unit', 'Used in %(count)s units',
+                    'Used in {count} unit', 'Used in {count} units',
                     count
                 );
             }
 
-            return interpolate(message, { count: count }, true);
-        },
-
-        getOutlineAnchorMessage: function () {
-            var message = _.escape(gettext(
-                    /*
-                        Translators: 'outlineAnchor' is an anchor pointing to
-                        the course outline page.
-                    */
-                    'This content group is not in use. Add a content group to any unit from the %(outlineAnchor)s.'
-                )),
-                anchor = str.sprintf(
-                    '<a href="%(url)s" title="%(text)s">%(text)s</a>',
-                    {
-                            url: this.model.collection.parents[0].outlineUrl,
-                            text: _.escape(gettext('Course Outline'))
-                    }
-                );
-
-            return str.sprintf(message, {outlineAnchor: anchor});
```

Comment:
See above in this file-- instead of outlineAnchorMessage being passed to the template (this was text that contained HTML), only courseOutlineUrl is passed to the template. The template does the necessary escaping, which allows the text portion of the message to be properly escaped. What was here in code was pretty much right, but in general we want to escape as late as possible.